tests/integration/test_arrowflight_interface/configs/grpc_config.xml

```diff
@@ -0,0 +1,7 @@
+<clickhouse>
+    <grpc replace="replace">
+        <enable_ssl>true</enable_ssl>
+        <ssl_cert_file>/etc/clickhouse-server/config.d/cert.pem</ssl_cert_file>
+        <ssl_key_file>/etc/clickhouse-server/config.d/key.pem</ssl_key_file>
+    </grpc>
```

Comment:
`grpc` and `arrowflight` are different protocols in ClickHouse. I mean it seems better to use for `arrowflight` its own section in the config, not `<grpc>`.

Reply:
fixed